components/alibi-detect-server/Dockerfile

```diff
@@ -70,4 +70,7 @@ COPY --from=builder /opt/conda /opt/conda
 COPY --from=builder /usr/bin/rclone /usr/bin/rclone
 COPY --from=builder /licenses /licenses
 
+# This is to have writable numba cache directory
+ENV NUMBA_CACHE_DIR /tmp/numba-cache
```

Comment:
is this something introduced with new alibi-detect version?

Reply:
yes, there is new dependency on numba and numba's default cache directory is not writable which lead to failures